File: rei-do-chopp-site/src/app/restockings/models/responses/restocking-response.ts
import { RestockingProductResponse } from './restocking-product-response';
import { RestockingAdditionalFeeResponse } from './restocking-additional-fee-response';

export interface RestockingResponse {
  Id: number;
  Date: Date;
  UserName: string;
  RestockingAdditionalFees: RestockingAdditionalFeeResponse[];
  RestockingProducts: RestockingProductResponse[];
}

File: rei-do-chopp-api/ReiDoChopp.DataTransfer/Restockings/Responses/RestockingResponse.cs
using ReiDoChopp.DataTransfer.RestockingAdditionalFees.Response;
using ReiDoChopp.DataTransfer.RestockingProducts.Response;


namespace ReiDoChopp.DataTransfer.Restockings.Response
{
    public class RestockingResponse
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string UserName { get; set; }
        public IList<RestockingAdditionalFeeResponse> RestockingAdditionalFees { get; set; }
        public IList<RestockingProductResponse> RestockingProducts { get; set; }
    }
}
